### DISCUSSION TITLE
Different Models for Anytrading

### DISCUSSION THREAD

**Discussion by abhishekjolad** [opendilab/DI-engine#824] [Q&A]
How can I replace the DQN model from the demo code with a PPO model and an A2C model?

**Comment by PaParaZz1** [comment]
Of course sure! You can try other model and policy defined in DI-engine. For more concrete examples, you can refer to [here](https://github.com/opendilab/DI-engine/tree/main/ding/example).

**Reply by abhishekjolad** [reply]
I managed to modify the example code but everytime I run the model, it always shorts the stock and returns a 0% return. How do I fix it?
![Screenshot 2024-08-04 192826](https://github.com/user-attachments/assets/16f86036-fc79-460c-b84a-a23ac43c22fc)
![Screenshot 2024-08-04 192916](https://github.com/user-attachments/assets/2b8ee853-6fdf-4ac2-aa19-e561973a032a)
